README.adoc

```diff
@@ -174,6 +174,15 @@ See <<message, Message>>.
 | The label `Perso` does not exist.
 |===
 
+[[severity-level]]
+==== Severity level
+
+The severity can be one of the following:
+
+* `WARNING`: The query may have issues. Please review it. We might be able to give you a suggestion on how to improve it.
+* `INFORMATION`: The query may have issues, even though it is correct. The provided information can still be helpful.
```

Comment:
I would like to also add that we are generally more certain the query have issues for WARNING compar.ed to INFORMATION. Maybe @LinneaAndersson has some feedback too

Reply:
How about:
* `WARNING`: It is about queries/commands that may have structural, syntactic, semantic, or spelling issues that prevent them from being executed in an optimal way. We need to give example(s) and suggestion(s) for improvement.
* `INFORMATION`: It is about queries/commands that are correct but may have some performance issues or that have no effect because they try to do something already done. Depending on the case, we may give example(s) and suggestion(s) for improvement.

`Neo.ClientNotification.Database.HomeDatabaseNotFound` is the only status code that deviates from the above definition for `INFORMATION`.

Reply:
I think that is better 👍